src/main/java/net/imagej/ops/image/ImageNamespace.java

```diff
@@ -447,13 +373,10 @@ public Object scale(final Object... args) {
 
 	/** Executes the "scale" operation on the given arguments. */
 	@OpMethod(op = net.imagej.ops.image.scale.ScaleImg.class)
-	public <T extends RealType<T>> Img<T> scale(final Img<T> in,
-			final double[] scaleFactors,
+	public <T extends RealType<T>> Img<T> scale(final Img<T> in, final double[] scaleFactors,
 			final InterpolatorFactory<T, RandomAccessible<T>> interpolator) {
 		@SuppressWarnings("unchecked")
-		final Img<T> result = (Img<T>) ops().run(
-				net.imagej.ops.Ops.Image.Scale.class, in, scaleFactors,
-				interpolator);
+		final Img<T> result = (Img<T>) ops().run(net.imagej.ops.Ops.Image.Scale.class, in, scaleFactors, interpolator);
 		return result;
```

Comment:
now you've formatted all the other methods. please undo the formatting for everything else and just add your changes?